app/src/main/java/tttTDD/Java/Game.java

```diff
@@ -58,7 +58,7 @@ public boolean performMove(int playerInput) {
         int input = !currentPlayer.isComputer() ? playerInput : board.getFirstAvailableMove();
 
         if (board.updateBoard(input, currentPlayer.getMarker())) {
-            gameCompleted = gameHasWinner();
+            gameCompleted = gameHasFinished();
```

Comment:
Nice rename there. gameHasWinner becoming gameHasFinished showed the thought put into the refactor of the method to match the changes that happened within it.